compiler/src/main/kotlin/edu/cornell/cs/apl/viaduct/backends/cleartext/CleartextCodeGenerator.kt

```diff
@@ -415,4 +416,6 @@ class CleartextCodeGenerator(context: CodeGeneratorContext) :
         }
         return receiveBuilder.build()
     }
+
+    override fun setup(protocol: Protocol): Iterable<PropertySpec> = listOf()
```

Comment:
You should add this to `AbstractCodeGenerator` so the old code generators don't have to change. Make sure to use `open`:

```kotlin
open override fun setup(protocol: Protocol): Iterable<PropertySpec> = listOf()
```

Reply:
I added the open keyword and IntelliJ greyed it out, wondering why?

Reply:
That's right, `overrides` are open by default. You need `open` is you declare a method in an abstract class.